config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -1007,3 +1007,52 @@ periodics:
   annotations:
     testgrid-dashboards: google-gce
     testgrid-tab-name: soak-gci-gce-1.12
+
+  - name: pull-kubernetes-e2e-gce-nodeoutofservicedetach
```

Comment:
@xing-yang -- Not sure, what should the approach to run this test in this case. This job only runs the node out of service feature scale test. Is this the right approach?